packages/signers/src/wasm.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./index";
```

Comment:
```suggestion
export { type Signer } from "./signer";
```

Lets just export whats needed for standalone WASM usage, if they need anything from the standard entrypoint they can just import that as well easy enough. Since we're already splitting entrypoints this will just make it so the JS bundle size is smaller in the case only the WASM based signer is needed